src/ring_theory/perfection.lean

```diff
@@ -186,7 +186,7 @@ lemma of : perfection_map p (perfection.coeff R p 0) :=
 mk' (ring_equiv.refl _) $ (equiv.apply_eq_iff_eq_symm_apply _).2 rfl
 
 /-- For a perfect ring, it itself is the perfection. -/
-def id [perfect_ring R p] : perfection_map p (ring_hom.id R) :=
+lemma id [perfect_ring R p] : perfection_map p (ring_hom.id R) :=
```

Comment:
Maybe `perfection_map` should be renamed to `is_perfection_map` (in another PR) to make it more clear that it's `Prop`-valued.

Anyways, the linter is happy now, so let's try again.

bors r+

And just in case...
bors d+